src/core/client/admin/components/ModerateCard/LinkDetailsContainer.tsx

```diff
@@ -0,0 +1,84 @@
+import { Localized } from "fluent-react/compat";
+import React, { FunctionComponent } from "react";
+import CopyToClipboard from "react-copy-to-clipboard";
+
+import { graphql, withFragmentContainer } from "coral-framework/lib/relay";
+import { Button, Icon } from "coral-ui/components";
+
+import { LinkDetailsContainer_comment } from "coral-admin/__generated__/LinkDetailsContainer_comment.graphql";
+import { LinkDetailsContainer_settings } from "coral-admin/__generated__/LinkDetailsContainer_settings.graphql";
+
+import styles from "./LinkDetailsContainer.css";
+
+interface Props {
+  comment: LinkDetailsContainer_comment;
+  settings: LinkDetailsContainer_settings;
+}
+
+const LinkDetailsContainer: FunctionComponent<Props> = ({
+  comment,
+  settings,
+}) => {
+  return (
+    <>
+      <div className={styles.label}>
+        <Localized id="moderate-linkDetails-label">
+          Copy link to this comment
+        </Localized>
+      </div>
+      <div className={styles.buttonContainer}>
+        <CopyToClipboard text={`${comment.story.url}?commentID=${comment.id}`}>
```

Comment:
Should use `getURLWithCommentID` instead.

Reply:
Makes sense!